x-pack/plugins/upgrade_assistant/public/application/lib/es_deprecation_errors.ts

```diff
@@ -25,8 +25,7 @@ const i18nTexts = {
   upgradedMessage: i18n.translate(
     'xpack.upgradeAssistant.esDeprecationErrors.upgradedWarningMessage',
     {
-      defaultMessage:
-        'Your configuration is up to date. Kibana and all Elasticsearch nodes are running the same version.',
+      defaultMessage: 'All Elasticsearch nodes have been upgraded.',
```

Comment:
> Can you help me understand this? What does it mean? Why is it considered an error? What do we expect the user to do once they see this message?

I'll work with @debadair on the copy here. In UA, there is [existing logic ](https://github.com/elastic/kibana/blob/master/x-pack/plugins/upgrade_assistant/server/lib/es_version_precheck.ts)that checks all node versions in the cluster. We return a 426 if there is ever a version mismatch between Kibana and any running nodes. If all nodes don't match with Kibana, we assume they have been upgraded.

For the 6-7 upgrade, the copy was: "All Elasticsearch nodes have been upgraded. You may now upgrade Kibana." I don't think we can say this now though, since there may be Kibana deprecations that need to be addressed as well.